tools/test-proxy/Azure.Sdk.Tools.TestProxy.Tests/TestHelpers.cs

```diff
@@ -191,7 +195,11 @@ public static string DescribeTestFolder(Assets assets, string[] sampleFiles, str
                 {
                     if (fullPath.EndsWith("assets.json"))
                     {
-                        assetsJsonPath = fullPath;
+                        // write assets json if we were passed content
```

Comment:
This simple change allows us to write multiple assets.jsons in the same testfolder. This allows us to run restores against multiple assets.jsons.